src/utils/send-message-formatter.js

```diff
@@ -0,0 +1,11 @@
+function formatSendMessage({ title, description, userName }) {
+  return `
+    # ${title}
+
+    ${description}
+
+    > Anuncio: ${userName}
+    `;
```

Comment:
estaba pensando en algo asi para que los campos puedan ser opcionales
```suggestion
    ${title ? `# ${title}` : ''}
    ${description ? `\n\n${description}` : ''}
    > Anuncio: ${userName}
    `;
```

Reply:
![image](https://github.com/user-attachments/assets/d204db39-e47c-4736-9ddd-9ff842b35a3a)
The change is ready